code/modules/mining/abandonedcrates.dm

```diff
@@ -23,16 +23,10 @@
 /obj/structure/closet/crate/secure/loot/proc/generate_loot()
 	var/loot = rand(1, 100)
 	switch(loot)
-		if(1 to 5) // Common things go, 5%
-			new/obj/item/weapon/reagent_containers/food/drinks/bottle/rum(src)
-			new/obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey(src)
-			new/obj/item/weapon/reagent_containers/food/snacks/grown/ambrosiadeus(src)
-			new/obj/item/weapon/flame/lighter/zippo(src)
-		if(6 to 10)
```

Comment:
Or these?

Reply:
because who cares

also i needed more rng slots so I decided to get rid of 2 shitty mining related drops so that kinetic accelerators may replace them